site/en/blog/top-layer-devtools/index.md

```diff
@@ -31,8 +31,7 @@ The [top layer](https://fullscreen.spec.whatwg.org/#new-stacking-layer) can be d
 Multiple elements can be inside the top layer at the same time. When that happens, they stack on top of each other, the last one on top. In other words, all of the top layer elements are placed in a *last in, first out* (LIFO) stack in the top layer.
 
 The `<dialog>` element is not the only element that the browser renders into a top layer. Currently, the top layer elements are:
-[popups](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/user_interface/Popups)
-[modal dialogs](https://developer.mozilla.org/docs/Web/HTML/Element/dialog) elements in a [fullscreen mode](https://developer.mozilla.org/docs/Web/API/Fullscreen_API).
+[popups](https://open-ui.org/components/popup.research.explainer), [modal dialogs](https://developer.mozilla.org/docs/Web/HTML/Element/dialog), and elements in a [fullscreen mode](https://developer.mozilla.org/docs/Web/API/Fullscreen_API).
```

Comment:
One nit: this should be "pop-ups" with a dash.

Reply:
Either with or without a dash is appropriate in this instance, though the new link references the "Pop Up API" and "pop-ups". Our style guide says to [avoid usage](https://developers.google.com/style/word-list#pop-up) however it is appropriate when it's the name of the component.

Reply:
In this case, it's material. Please see this discussion: https://github.com/openui/open-ui/issues/546#issuecomment-1158190204

The name of the API was purposely changed to "the pop-up API" so that "pop" and "up" are two words, so that the IDL attribute can be called `popUp` with a capital "U", so we don't break the web. Ugh!

So here, ideal would be "pop-ups", but "pop ups" would also be ok. But not-ok is "popups". Sorry.

Reply:
Thanks for the explanation! I've updated the PR to reflect this point.